src/rez/plugin_managers.py

```diff
@@ -201,9 +176,55 @@ def load_plugins(self):
                         traceback.print_exc(file=out)
                         print_debug(out.getvalue())
 
-            # load config
-            data, _ = _load_config_from_filepaths([os.path.join(path, "rezconfig")])
-            deep_update(self.config_data, data)
+    def load_plugins_from_entry_points(self):
+        if sys.version_info.minor >= 8:
+            from importlib.metadata import entry_points
+        else:
+            from importlib_metadata import entry_points
```

Comment:
This will force us to vendor `importlib-metadata` (which is fine since it's an APACHE-2.0 licensed).